pulsar-zookeeper-utils/src/main/java/com/yahoo/pulsar/zookeeper/ZooKeeperDataCache.java

```diff
@@ -143,7 +143,7 @@ public void process(WatchedEvent event) {
         }
     }
 
-    public void close() {
+    public synchronized void close() {
```

Comment:
Since clear() is not synchronized, what benefit we will get by making close() synchronized?